### DISCUSSION TITLE
How to deploy nuxt 3 SSR to vercel via self hosted gitlab CI

### DISCUSSION THREAD

**Discussion by marysmech** [nuxt/nuxt#16817] [Questions]
Im really stucked on nuxt 3 SSR deploy to vercel. I have already sumed up my problem here https://stackoverflow.com/questions/73755848/nuxt-3-ssr-deploy-on-vercel-via-self-hosted-gitlab-ci 

According to docs it should work out of the box. I tried to change `OUTPUT DIRECTORY` to diffent location (like `.output/public`) and after change deploy its fine byt I get 404 when i reach the page. I think im missing something but i have really no idea what. 

Any advice would be appreciated.

**Comment by sevillaarvin** [comment]
Not sure but I think it's temporarily unsupported at the moment: https://github.com/nuxt/starter/issues/23#issue-1220390472

I tried following the docs but when I install the template `npx nuxi init -t v3-vercel` I get:

```
└──╼ $ npx nuxi@latest init -t v3-vercel
Nuxi 3.0.0-rc.11                                                                                                                                                                     23:56:35

 ERROR  (node:72890) ExperimentalWarning: The Fetch API is an experimental feature. This feature could change at any time                                                            23:56:35
(Use `node --trace-warnings ...` to show where the warning was created)


 ERROR  Failed to download template from registry: Failed to download v3-vercel template info from https://raw.githubusercontent.com/nuxt/starter/templates/templates/v3-vercel.json: 404 Not Found

  at node_modules/nuxi/dist/chunks/init.mjs:13269:11
  at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  at async downloadTemplate (node_modules/nuxi/dist/chunks/init.mjs:13268:20)
  at async Object.invoke (node_modules/nuxi/dist/chunks/init.mjs:13336:15)
  at async _main (node_modules/nuxi/dist/cli.mjs:50:20)
```

Docs: https://v3.nuxtjs.org/guide/deploy/providers/vercel#vercel

**Comment by bot08** [comment]
This may help u [#7773](https://github.com/nuxt/framework/issues/7773)
